executor/executor.go

```diff
@@ -134,19 +135,20 @@ func (e *baseExecutor) Close() error {
 	return nil
 }
 
-// Schema implements the Executor Schema interface.
+// Schema returns current baseExecutor's schema. If its value is nil, then create a new one.
```

Comment:
the current

Reply:
actually, I feel like  either "the baseExecutor's schema" or "current baseExecutor's schema". "the current" make me thinking it's duplicate meaning such as "In addition, we also implement...."